units/patch_intent.go

```diff
@@ -499,9 +499,8 @@ func (j *patchIntentProcessor) buildTasksandVariants(patchDoc *patch.Patch, proj
 func setTasksToPreviousFailed(patchDoc, previousPatch *patch.Patch, project *model.Project) error {
 	var failedTasks []string
 	for _, vt := range previousPatch.VariantsTasks {
-		tasksInProjectVariant := project.FindTasksForVariant(vt.Variant)
 		var tasks []string
```

Comment:
Unrelated nit but can we remove this declaration? Seems like the := at 503 should handle it so I'm guessing this is outdated